app/services/login/user_verifier.rb

```diff
@@ -137,6 +138,17 @@ def attempt_secondary_idme_identifier
       Rails.logger.info("[Login::UserVerifier] Attempting alternate type=#{type}  identifier=#{identifier}")
     end
 
+    def linked_user_verification_is_locked?
+      return false unless existing_user_account &&
+                          UserVerification.where(user_account: existing_user_account).any? do |v|
+                            v.send(type).present? && v.locked
+                          end
+
+      Rails.logger.info('[Login::UserVerifier] Locked UserVerification created ' \
```

Comment:
Oh yeah, looking at this deeper, this is in a transaction, so we can't have a log in the transaction. If you look at `deprecated_log`, `user_account_mismatch_log`, `new_user_log` you'll see other examples. In fact, I wonder if the value of locked should just be stored as a class variable and just used in the `new_user_log`. (maybe include identifier as well, so `new_user_log= "[Login::UserVerifier] New VA.gov user, type=#{login_type}, broker=#{auth_broker}, identifier=#{identifier}, locked=#{locked}"`